gravity.sh

```diff
@@ -35,8 +35,9 @@ localList="${piholeDir}/local.list"
 VPNList="/etc/openvpn/ipp.txt"
 
 piholeGitDir="/etc/.pihole"
-gravityDBfile="${piholeDir}/gravity.db"
-gravityTEMPfile="${piholeDir}/gravity_temp.db"
+gravityDBfile_default="${piholeDir}/gravity.db"
+# GRAVITYDB may be overwritten by source pihole-FTL.conf below
+GRAVITYDB="${gravityDBfile_default}"
```

Comment:
What variable is set in the `source`? If it's `GRAVITYDB` then:

```suggestion
GRAVITYDB="${GRAVITYDB:-${piholeDir}/gravity.db}"
```
https://wiki.bash-hackers.org/syntax/pe#use_a_default_value

```bash
dan@Viking-1:~$ piholeDir="/pi"
dan@Viking-1:~$ GRAVITYDB="${GRAVITYDB:-${piholeDir}/gravity.db}"
dan@Viking-1:~$ echo $GRAVITYDB
/pi/gravity.db
dan@Viking-1:~$ unset GRAVITYDB
dan@Viking-1:~$ echo $GRAVITYDB

dan@Viking-1:~$ GRAVITYDB="/bar/gravity.db"
dan@Viking-1:~$ echo $GRAVITYDB
/bar/gravity.db
```

Reply:
I store them in two distinct locations so we can easily decide if we need to edit the SQL script and/or print a note that a custom location is in use, see

![Screenshot from 2021-03-24 08-08-43](https://user-images.githubusercontent.com/16748619/112269206-59790680-8c78-11eb-92d9-57655b8f1f2a.png)
![Screenshot from 2021-03-24 08-08-33](https://user-images.githubusercontent.com/16748619/112269149-49612700-8c78-11eb-8b79-fbda85b9d1df.png)